uninstall.sh

```diff
@@ -1,31 +1,25 @@
 #!/usr/bin/env bash
-if [ -z "$BASH_IT" ];
-then
-  BASH_IT="$HOME/.bash_it"
-fi
+: "${BASH_IT:=${HOME?}/.bash_it}"
 
-case $OSTYPE in
-  darwin*)
-    CONFIG_FILE=.bash_profile
-    ;;
-  *)
-    CONFIG_FILE=.bashrc
-    ;;
-esac
+CONFIG_FILE=".bashrc"
+if [[ ! -e ~/.bashrc && -e ~/.bash_profile ]]; then
+	# legacy Mac or WSL or just no backup file
+	CONFIG_FILE=".bash_profile"
+fi
 
 BACKUP_FILE=$CONFIG_FILE.bak
 
-if [ ! -e "$HOME/$BACKUP_FILE" ]; then
-  echo -e "\033[0;33mBackup file $HOME/$BACKUP_FILE not found.\033[0m" >&2
+if [[ ! -e "${HOME?}/$BACKUP_FILE" ]]; then
+	echo -e "\033[0;33mBackup file ${HOME?}/$BACKUP_FILE not found.\033[0m" >&2
```

Comment:
you can use colors correctly here as well

Reply:
Not without sourcing `lib/colors`, but since we're *uninstalling* it's entirely possible the installation is broken so we should just keep everything self-contained in this one script.

Reply:
you can add a comment on top of this file explaining this very logic

Reply:
Pushed.